### PR TITLE
Add tests for new sell tab landing page (including fixing several tracking bugs)

### PR DESCRIPTION
**Pull request by pepopowitz**
Collaboration with @xtina-starr 

This PR adds some test coverage to the new sell tab.

It also updates the `RecentlySold` component to use cohesion for tracking, and fixes several tracking bugs:
* we were using the wrong `context_module` for tracking taps of the footer button
* we were not including the `destination_screen_owner_id` or `destination_screen_owner_slug` when tracking artist list taps
* we were not including the `context_screen_owner_type` when tracking recently sold taps

### CODE REVIEW

File: src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/Footer.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import styled from "styled-components/native"
 import { TextContainer } from "./TextContainer"
 
 const consignArgs: TappedConsignArgs = {
-  contextModule: ContextModule.sellHeader,
+  contextModule: ContextModule.sellFooter,
```
Comment (brainbicycle) [inline src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/Footer.tsx:8]: Nice 👍 glad cohesion is getting some traction in the app

Comment (jonallured) [reply src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/Footer.tsx:8]: Damn, thanks for fixing my mistake!! 😍

Comment (pepopowitz) [reply src/lib/Scenes/Consignments/v2/Screens/ConsignmentsHome/Components/Footer.tsx:8]: @jonallured *our* mistake. I missed it in review the first time, too.

**Comment by jonallured** [comment]
LGTM!! 🎉 #mergeongreen

**Pull request merged**